cli-output/src/cli_output.rs

```diff
@@ -1766,13 +1767,15 @@ impl CliFees {
         blockhash: Hash,
         lamports_per_signature: u64,
         last_valid_slot: Option<Slot>,
```

Comment:
`last_valid_slot` will always return None as of this change. Does this seem too breaky?

Reply:
🤔 My gut is that this will be ok,  it only affects `solana fees--output json`, and I'd be surprised if anybody relies on `lastValidSlot` from there